### PR TITLE
Added new automated UI test for Arquillia Cruiser View

### PR DESCRIPTION
**Pull request by ldimaggi**
This PR is for the master and 4.3.x branches - thanks!

### CODE REVIEW

File: tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+/**
+ * Arquillia Cruiser View tests 
+ * 
+ * @author Len DiMaggio
+ *
+ */
+
+public class BasicArquilliaCruiserTest {
+
+	private static Logger log = new Logger(HTML5Parameterized.class);
```
Comment (rhopp) [inline tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java]: I guess you don't want to log as HTML5Parameterized.class, right? ;-)

Comment (rhopp) [inline tests/org.jboss.tools.arquillian.ui.bot.test/src/org/jboss/tools/arquillian/ui/bot/test/cruiserView/BasicArquilliaCruiserTest.java]: Changing this to BasicArquillianCruiserTest.class would make change in tests/org.jboss.tools.central.ui.bot.test/META-INF/MANIFEST.MF irrelevant.

**Comment by ldimaggi** [comment]
Argh! Thanks Radim - I just did an amended commit.

**Comment by rhopp** [comment]
Pushed to master and 4.3.x branch. Closing.

**Pull request closed without merging**